src/cluster.c

```diff
@@ -120,13 +120,12 @@ void listClusterNodeDestructor(void *val) { freeValkeyClusterNode(val); }
 
 void listClusterSlotDestructor(void *val) { cluster_slot_destroy(val); }
 
-unsigned int dictSdsHash(const void *key) {
+unsigned long int dictSdsHash(const void *key) {
```

Comment:
Is there a risk this and other functions are defined in valkey too with a different hash function under the hood? Any risk we or valkey will use the wrong one if both are statically linked in the same binary, or will we get an link-time error if that happens?